owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/utils/Log_OC.java

```diff
@@ -134,9 +134,9 @@ public static void deleteHistoryLogging() {
         File folderLogs = new File(mFolder + File.separator);
         if(folderLogs.isDirectory()){
             String[] myFiles = folderLogs.list();
-            for (int i=0; i<myFiles.length; i++) {
-                File myFile = new File(folderLogs, myFiles[i]);
-                myFile.delete();
+            for (String myFile1 : myFiles) {
```

Comment:
I'd call it myFile instead of myFile1.

Reply:
In the next line the name `myFile` is occupied.
Anyway `myXxxx` has now additional name value too (it's close to a number), so I gave them other names